src/fusor/models.py

```diff
@@ -22,6 +21,8 @@
     model_validator,
 )
 
+CURIE_REGEX = r"^\w[^:]*:.+$"
```

Comment:
This was defined somewhere else. I would recommend importing this rather than redefining.